src/plugins/vis_type_table/public/types.ts

```diff
@@ -49,3 +56,13 @@ export interface TableVisConfig extends TableVisParams {
   title: string;
   dimensions: Dimensions;
 }
+
+export interface FormattedColumn {
+  id: string;
+  title: string;
+  formatter?: ReturnType<DataPublicPluginStart['fieldFormats']['deserialize']>;
```

Comment:
I think this type should be exported from the Data plugin? (If not it def sounds like something that should be exported propertly).

Reply:
thx, done